openapi-cli/src/test/resources/generators/schema/ballerina/default_required_field_schema.bal

```diff
@@ -0,0 +1,5 @@
+public type Pet record {
+    int id;
+    string name;
+    string 'type?;
+};
```

Comment:
I'm not seeing any default value in the expected result in here. Am I missing something?

Reply:
If default values is specified in the required values we don't add default value to it